sigs/cncf-sigs.md

```diff
@@ -127,11 +142,13 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 
 ### Elections
 
-* The TOC nominates Chairs
-* Chairs are assigned following a 2/3 majority vote of the TOC
-* Terms last for 2 years but staggered such that at least 1 of the chairs is able to maintain continuity
-* The TOC and Chairs nominate Tech leads
-* Tech leads are assigned following a 2/3 majority vote of the TOC and a 2/3 majority vote of SIG Chairs
+* Chairs
+  * Nomination: TOC
+  * Vote: 2/3 majority of the TOC
+  * Terms last for 2 years but staggered such that at least 1 of the chairs is able to maintain continuity
+* Tech leads
+  * Nomination: TOC liaison or Chair
+  * Vote: 2/3 majority of the TOC and 2/3 majority vote of SIG Chairs
```

Comment:
Realised this is ambiguous - does it mean all the chairs from across all the SIGs, or just the chairs of the particular SIG in question? I think it should just be from the one SIG (because those SIG chairs know their community best, plus it might be hard to achieve the 2/3 votes across the whole SIG community).